src/signalrclient/callback_manager.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+#include<atomic>
+#include<unordered_map>
+#include<functional>
+#include<mutex>
+#include<cpprest\json.h>
+
+namespace signalr
+{
+    class callback_manager
+    {
+    public:
+        explicit callback_manager(const web::json::value& dtor_error);

```

Comment:
So I have this straight... this is a json value that's passed to callbacks to let them know they've been "cleared" right? I'd add a comment above the c'tor.